doc/protocol.rst

```diff
@@ -0,0 +1,270 @@
+GA4GH Wire Protocol Specification
+=================================
+
+Version 0.3 [draft]
+
+This document defines the GA4GH data exchange API Internet
+protocol.  It provides a restartable, streaming protocol with
+both ASCII and binary encoding formats.  It is designed to
+efficiently send a potentially long stream of small messages.
+
+The streaming protocol replaces the current GA4GH paging protocol.
```

Comment:
- The streaming protocol replaces the current GA4GH paging protocol.
+ This streaming protocol replaces the current GA4GH paging protocol.